_layouts/blog-home.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+
+{% include head.html %}
+
+<body>
+    {% include header-extended.html %}
+    <section class="usa-section bg-blue-80v">
+        <div class="grid-container">
+            <h1 class="font-heading-3xl text-base-lightest">{{ page.title }}</h1>
+            <p class="font-alt-xl text-base-lightest">{{ page.description }}</p>
+        </div>
+    </section>
+
+    <main id="main-content">
+        <div class="grid-container padding-y-6">
+            <div class="grid-row grid-gap">
+                <div class="tablet:grid-col-4 tablet:order-2">
+                    {% include search-small.html %}
+                    <h2 class="margin-top-3">Categories</h2>
```

Comment:
Change from Categories to Tags. (See comment on line 45 of .md file for more context.)